systemtest/src/test/java/io/strimzi/systemtest/utils/KafkaVersionUtilsTest.java

```diff
@@ -4,15 +4,17 @@
  */
 package io.strimzi.systemtest.utils;
 
-import org.junit.jupiter.api.Test;
+import io.strimzi.systemtest.annotations.ParallelSuite;
+import io.strimzi.systemtest.annotations.ParallelTest;
 
 import java.util.List;
 
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+@ParallelSuite
 public class KafkaVersionUtilsTest {
 
-    @Test
```

Comment:
Does this really run with the System Tests? Should this really be `ParallelTest`?

Reply:
Yes and yes it's ok that this test is parallel (by default all tests are serial)